pkgs/by-name/bc/bc-ur/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, fetchpatch
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bc-ur";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "BlockchainCommons";
+    repo = "bc-ur";
+    rev = finalAttrs.version;
+    hash = "sha256-j7nmAZH1OL7R5H3jqQomg7kwPOvIHMqrfSk7mq/f7Cg=";
+  };
+
+  patches = [
+    (fetchpatch {
```

Comment:
Why?

Reply:
Otherwise it doesn't build, Makefile passes nonexistent flags to compiler, some imports are straight up missing.
This patch comes from Feather itself and fixes basically everything above + adds building through CMake instead of their custom Makefiles

Reply:
```suggestion
    # Fix build on gcc13, add CMakeList.txt
    (fetchpatch {
```